### ISSUE TITLE
Invoke `parse_module` command on each save/modify action

### ISSUE THREAD

**Issue by magniff** [alygin/vscode-tlaplus#163]
Repeatedly hitting this "parse_module" command is a bit annoying, why not automate?

**Comment by alygin** [comment]
@magniff, there's an article in the project's Wiki that describes how to configure VS Code to parse modules automatically: [Automatic Module Parsing](https://github.com/alygin/vscode-tlaplus/wiki/Automatic-Module-Parsing).

Unfortunately, the VS Code API doesn't allow an extension to set such a setting programmatically or via a contribution point. At least, I didn't find how to do that.

So, if the setting, described in the article, works for you, I'm inclined to close this issue.

**Comment by magniff** [comment]
Thanks for your response! Basically I'm asking whether we can make it `true` _by default_, I guess it is set somewhere [around that line](https://github.com/alygin/vscode-tlaplus/blob/master/package.json#L314)?

**Comment by alygin** [comment]
@magniff, you're right, this is the place where we could define the default value for the setting. Moreover, in some very early versions, it was set to `true`, but then I removed it because most popular extensions didn't do such things, it was a common practice to provide only basic functionality by default and let the user decide what features he or she really needs and switch them on manually.

But it looks like things have changed since then, and now most extensions perform various actions on file save by default. So, yeah, it looks reasonable to make it `true` by default again.
